### PR TITLE
Updated section 1 to replace ku with bdk-cli

### PR DESCRIPTION
**Pull request by i5hi**

### CODE REVIEW

File: content/blog/2020/descriptors_in_the_wild.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ that the two parties had to agree:
...
-addresses to produce with a key or a serie of keys.
+addresses to produce with a key or a series of keys.
 
-[PSBT] is instead the standard protocol used to create transaction and to enrich 
+[PSBT] is instead the standard protocol used to create a transaction and to enrich 
 it with the necessary signatures and other components, to make it valid and complete.
 
 Together they provide a common ground to create and use a multi signature 
-infrastructure in a eterogeneous environment and this is what I have put 
+infrastructure in an eterogeneous environment, and this is what I have put
```
Comment (notmandatory) [inline content/blog/2020/descriptors_in_the_wild.md]: should also correct "a eterogeneous" -> "a heterogeneous"

File: content/blog/2020/descriptors_in_the_wild.md [hunk 2]
```diff
@@ -42,10 +42,10 @@ transaction.
 ## The role of Descriptors
 
 If Alice and Bob cannot agree on the software to use, to monitor the same financial 
-situation, the two software must control and produce exactly the same series
+situation, the two softwares must control and produce exactly the same series
```
Comment (notmandatory) [inline content/blog/2020/descriptors_in_the_wild.md]: sorry plural of software is software.. English grammar is weird. 
https://thewordcounter.com/plural-of-software/

File: content/blog/2020/descriptors_in_the_wild.md [hunk 3]
```diff
@@ -113,45 +111,43 @@ For Bitcoin Core (Alice):
...
 
-export core_xpub_84=$(ku -j -s 84H/0H/0H $core_xprv |jq -r '.public_version')
-export core_fingerprint=$(echo $core_key|jq -r '.fingerprint')
+export core_xpub_84=$(bdk-cli key derive --path m/84h/0h/0h --xprv $core_xprv | jq -r '.xpub' | rev | cut -c 3- | rev)
 
 # Now I build the xpubs (one for receiving and one for the change) 
 # together with informations about the derivation path to be communicated 
 # to BDK wallet's owner (Bob).
 
-export core_xpub_84_for_rec_desc="[$core_fingerprint/84'/0'/0']$core_xpub_84/0/*"
```
Comment (notmandatory) [inline content/blog/2020/descriptors_in_the_wild.md:131]: To keep the example more or less the same you can use the `bdk-cli key derive` command to derive the pubkey for this descriptor

**Review by notmandatory — CHANGES_REQUESTED** [review]
Thanks for doing this update, I have a few suggestions in-line.

File: content/blog/2020/descriptors_in_the_wild.md (hunk 2 above)
Comment (gabridome) [reply content/blog/2020/descriptors_in_the_wild.md]: So I guess I won't correct it. I will make a PR in your repository so you can apply a new PR.

**Comment by i5hi** [comment]
Updated with all the suggested changes.

**Comment by notmandatory** [comment]
I double checked the new `bdk-cli` parts and they LGTM. @gabridome is it OK for you if we merge these changes? Do you think @vmenond should add a note at the end with something like "Updated April 2021 to use `bdk-cli` instead of `ku`" or something like that... ?

**Comment by gabridome** [comment]
Absolutely.
Ku is a wonderful instrument, but if we can avoid installing it just to
derive the keys, it is more than fine with me.

Il Mar 20 Apr 2021, 17:49 Steve Myers ***@***.***> ha scritto:

**Comment by i5hi** [comment]
So would it be required to add 'Updated April,2021' ?

**Comment by notmandatory** [comment]
> So would it be required to add 'Updated April,2021' ?

Well I guess we don't have to the changes are in the commit history for anyone interested :slightly_smiling_face: I'll merge your changes now, good job!

**Review by notmandatory — APPROVED** [review]

**Pull request merged**